src/components/App.js

```diff
@@ -15,20 +9,9 @@ class App extends Component {
 
   render() {
     return (
-      <BrowserRouter>
-        <div>
-          <ul>
-            <li>
-              <Link to="/login" className="login"> Login </Link>
-            </li>
-          </ul>
-          <Switch>
-            <Route exact path="/" component={Home} />
-            <Route path="/login" component={Login} />
-            <Route component={notFound} />
-          </Switch>
-        </div>
-      </BrowserRouter>
+      <div>
+        <h1 className="coloring">Welcome to Authors Haven</h1>
+      </div>
```

Comment:
@NicholusMuwonge , what was the reason for removing the routes!?

Reply:
It was a conflict and Initially, I had demonstrated it this way.
But I can work around it with the routes

Reply:
I have reinstated the routes and Implemented it that way.

Reply:
Create a `routes` folder with an `index` file containing the routes to the home page, login, and any other route.
Call the routes index class/functional class in the `App.js` (think of how the app will be set up, not just your task)

Reply:
I have reacted to this feedback kindly please look into it.